core/templates/installation.php

```diff
@@ -1,11 +1,13 @@
 <?php
 script('core', 'install');
+
+$urlGenerator = $_['urlGenerator'];
 ?>
 <input type='hidden' id='hasMySQL' value='<?php p($_['hasMySQL']) ?>'>
 <input type='hidden' id='hasSQLite' value='<?php p($_['hasSQLite']) ?>'>
 <input type='hidden' id='hasPostgreSQL' value='<?php p($_['hasPostgreSQL']) ?>'>
 <input type='hidden' id='hasOracle' value='<?php p($_['hasOracle']) ?>'>
-<form action="index.php" method="post" class="guest-box install-form">
+<form action="<?php $urlGenerator->getAbsoluteURL("index.php") ?>" method="post" class="guest-box install-form">
```

Comment:
```suggestion
<form action="<?= $urlGenerator->getAbsoluteURL("index.php") ?>" method="post" class="guest-box install-form">
```

If you generate the url you should also echo it ;)

It should also work to remove the action attribute because the default is then the current page.